instrumentation/github.com/emicklei/go-restful/otelrestful/go.mod

```diff
@@ -9,15 +9,19 @@ require (
 	github.com/stretchr/testify v1.10.0
 	go.opentelemetry.io/contrib/propagators/b3 v1.35.0
 	go.opentelemetry.io/otel v1.35.0
+	go.opentelemetry.io/otel/exporters/stdout/stdouttrace v1.35.0
 	go.opentelemetry.io/otel/metric v1.35.0
+	go.opentelemetry.io/otel/sdk v1.35.0
```

Comment:
@pellared will this be considered a testing dependency? Will it be imported by everyone who uses this instrumentation?

It seems like it is being imported by `otelrestful/test`

Reply:
@MrAlias what came up from this question?